### ISSUE TITLE
Suggest to the client that neurons can be clicked on

### ISSUE THREAD

**Issue by neuroBazzi** [MetaCell/salk-interactive-atlas#373]
_When hovering over a cell, make it pop out to show that the cord is interactive._

What they mean here is that when the used hover over a neuron they do not know that they click on it and that this would open the tabs on the right with all the information. They are asking for a way to make it more obvious to the user that they can click on a neuron and that something will happen. We do no not need to implement exactly the 'pop out' that they ask but something that hints at the user that there is an interaction with the application. @syamace-metacell thought? cc @afonsobspinto @zsinnema

**Comment by syamace-metacell** [comment]
@neuroBazzi Agreed on this. Depending on the time constraints, there are a few ways we can do this. Ideally, the neurons would have a hover state and an active (when being clicked) state - how difficult will this be to implement? Otherwise, a text indicating that they could click on those neurons might do the trick.

**Comment by neuroBazzi** [comment]
@syamace-metacell the hover state sounds good to me, and it means that you can click on it. but I am not sure what the 'active' state would comunicate. how the 'hover' state would look like? same marker but bigger size? what would be the most efficient technical approach @afonsobspinto ?

**Comment by syamace-metacell** [comment]
@neuroBazzi 'active' state might be useful for seeing which population they've selected and is active on the details viewer. but might get confusing now that I'm looking at the implementation. hover state would either make the marker bigger like you mentioned or a border around it and/or a tooltip saying "click to view population details" and cursor change (to the cursor that shows that it's clickable). Though, making the marker bigger might look weird and I'm not sure how having the border around it will look like considering there's multiple markers in a population - yes maybe @afonsobspinto could advise the most efficient technical approach

**Comment by neuroBazzi** [comment]
Thanks Sya, I think we should keep in mind that there can be many neurons showing up very close together, sometimes also from different populations (user can activate as many as they want simultaneously) so I'm a little hesitant about the tooltip solution as I feel it would be a constant appear/disappear of the tooltip as the user hovers over a bunch of different neurons. I see your concern about the size change too tho so I'm not really sure what to suggest here. I think ideally it would be something that can go away at some point when the user 'learns' about it. along this line, maybe we could communicate the possibility of clicking on neurons elsewhere (not necessarily by modifying the markers themselves).what do you think?

**Comment by afonsobspinto** [comment]
I think changing the cursor style when we are hovering a neuron might be a good enough indication.

From a technical point of view, making it bigger, add a border or changing the cursor would take roughly the same amount of time (4h using geppetto-meta onHoverListeners).

We can also have some sort of tutorial/walk through component like the one that shows on [neuroscan](https://neuroscan.net/). I would need to investigate how that was implemented before giving an estimate but would take more time than any of the other options. cc @neuroBazzi

**Comment by neuroBazzi** [comment]
thank you Afonso, let's keep it simple for now and assume the cursor style change via Geppetto
